src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I had to bump this to netcoreapp2.0 since M.A.Diagnostics.Identity.Service targets net46/netcoreapp2.0. Any reason why this package is not targeting netstandard1.x? @javiercn